src/providers/authenticationProvider.ts

```diff
@@ -87,7 +88,11 @@ export class TerraformCloudAuthenticationProvider implements vscode.Authenticati
   private _onDidChangeSessions =
     new vscode.EventEmitter<vscode.AuthenticationProviderAuthenticationSessionsChangeEvent>();
 
-  constructor(private readonly secretStorage: vscode.SecretStorage, private readonly ctx: vscode.ExtensionContext) {
+  constructor(
```

Comment:
Can we remove the command execution from the constructor now, because `authProvider.onDidChangeSessions` takes care of it, too?

```ts
const org = this.ctx.workspaceState.get('terraform.cloud.organization', '');
if (org === '') {
  await vscode.commands.executeCommand('terraform.cloud.organization.picker');
}
```